### PR TITLE
Bulk Minting on Single Collection Page

### PR DESCRIPTION
**Pull request by kyawthetkt**

### CODE REVIEW

File: src/pages/collections/single/Page.js [hunk 1]
```diff
@@ -46,6 +61,76 @@ const Page = () => {
...
+            list = data.filter(
+                (item) =>
+                    item.get("tokenId") == null ||
+                    typeof item.get("tokenId") === "undefined"
+            );
+        }
+        if (selectedTab === "minted") {
+            list = data.filter((item) => item.get("tokenId") > -1);
+        }
+
```
Comment (jibrail-idris) [inline src/pages/collections/single/Page.js:76]: What's the timer for?

File: src/pages/collections/single/Page.js [hunk 2]
```diff
@@ -46,6 +61,76 @@ const Page = () => {
...
+        return () => clearTimeout(timer);
+    }, [data, selectedTab]);
+
+    const callBulkMint = async () => {
+        if (selectedData.length < 1) {
+            alert("No Item to mint.");
+            return false;
+        }
+
```
Comment (jibrail-idris) [inline src/pages/collections/single/Page.js:89]: Do we have a universal loading component?

File: src/pages/collections/single/Page.js (hunk 1 above)
Comment (kyawthetkt) [reply src/pages/collections/single/Page.js:76]: When setting new array state, we have to wait a few seconds to reflect changes.

File: src/pages/collections/single/Page.js (hunk 2 above)
Comment (kyawthetkt) [reply src/pages/collections/single/Page.js:89]: @jibrail-idris yes will fix it.

**Review by pracheebapate — APPROVED** [review]

**Comment by kyawthetkt** [comment]
@jibrail-idris @shakil-adv  @pracheebapate 
Sample Collection -> https://canvaslandnft-665d6--pr26-devcvl-264-8qit9lub.web.app/collection/alphabets

**Review by jibrail-idris — APPROVED** [review]

**Review by shakil-adv — APPROVED** [review]

**Pull request merged**